src/librustc/util/common.rs

```diff
@@ -27,13 +27,27 @@ pub const FN_OUTPUT_NAME: &'static str = "Output";
 #[derive(Clone, Copy, Debug)]
 pub struct ErrorReported;
 
+thread_local!(static TIME_DEPTH: Cell<usize> = Cell::new(0));
+
+/// Read the current depth of `time()` calls. This is used to
+/// encourage indentation across threads.
```

Comment:
This comment seems to be nonsensical given that TIME_DEPTH is a thread-local static. Perhaps it should be an AtomicUsize and shared across threads instead?